tests/pom.xml

```diff
@@ -107,6 +107,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Why is this necessary?

Reply:
This allows you to see the output of what is running in the docker/testcontainers when the tests are running. Not necessary, just nice to be able to see the logs.